invision-app/app/components/deleteMovie/deleteMovie.js

```diff
@@ -1,26 +1,24 @@
 import React from 'react';
 import './deleteMovie.scss';
-
-export class DeleteMovie extends React.Component {
-  constructor (props) {
-    super(props);
-  }
-
-  delete () {
-    this.props.deleteMovie(this.props.id);
+import PropTypes from 'prop-types';
+export function DeleteMovie(props){
+  function deleteMovie () {
+    props.deleteMovie(this.props.id);
```

Comment:
You should not use **this** to access to props

Reply:
Modified.